Formula/openmsx.rb

```diff
@@ -20,11 +19,18 @@ class Openmsx < Formula
     sha256 "86fff1a90fff96cb0398184ef7ebdeb804edda4c9d34ee5e7278159df64b10e3" => :high_sierra
   end
 
+  depends_on "alsa-lib"
```

Comment:
`on_linux do` like the other formulas depending on `alsa-lib` do

Reply:
Can you elaborate a bit? I did not get it.

IMO, I think we need to have a alsa formulae for osx, otherwise, it cannot be built :(

Reply:
`alsa-lib` is a library that targets the Linux kernel's audio system, so it is linux-only.
If `alsa-lib` is required by `openmsx` on linux, we can wrap it with `on_linux`.
If `openmsx` fails to compile without `alsa-lib`, either we find a way to disable that, or it means it cannot be used on macOS.

Reply:
PS: upstream provides binaries for macOS, so I am sure there is a way to build without alsa-lib